File: 3Week/14Day/java/Algo06/src/test/five.java
package test;

public class five {
	public int[] solution(int[] arr, int n) {
        int[] answer = new int[arr.length];
        boolean a = arr.length % 2 == 0; 
        for(int i = 0; i < arr.length; i++){
            if(i % 2 == 0){
                if(a){
                    answer[i] = arr[i];
                }else{
                    answer[i] = arr[i] + n;
                }
            }else{
                if(a){
                    answer[i] = arr[i] + n;
                }else{
                    answer[i] = arr[i];
                }
            }
        }
        return answer;
    }
}

File: 4Week/16Day/과제/five.py
def solution(numbers):
    answer = -100000001
    for i in range(len(numbers)):
        for j in range(i + 1, len(numbers)):
            print(numbers[i], numbers[j])
            if answer < numbers[i] * numbers[j]:
                answer = numbers[i] * numbers[j]
    return answer